tasks/mysql_secure_installation.yml

```diff
@@ -22,6 +22,7 @@
     - '::1'
     - '127.0.0.1'
     - 'localhost'
+  when: mysql_check_root_password_presence
```

Comment:
if you set it to _true_ and have different root password set up without .my.cnf file - this step will fail